src/features/fireWeather/components/graphs/TempRHGraph.tsx

```diff
@@ -89,22 +91,52 @@ const TempRHGraph: React.FunctionComponent<Props> = ({
 
         return { ...d, date }
       })
-      const recentHistoricModelValues = _recentHistoricModelValues.map(d => {
-        const date = d3Utils.storeDaysLookup(daysLookup, d.datetime)
-        const historicModel = {
-          date,
-          historicModelTemp: Number(d.temperature.toFixed(2)),
-          historicModelRH: Math.round(d.relative_humidity)
-        }
-        // combine with existing weather values
-        weatherValueByDatetime[d.datetime] = {
-          ...weatherValueByDatetime[d.datetime],
-          ...historicModel
-        }
-        allDates.push(date)
+      const recentHistoricModelValues = _recentHistoricModelValues
+        .filter(d => {
+          return d.temperature !== null
+        })
+        .map(d => {
+          const date = d3Utils.storeDaysLookup(daysLookup, d.datetime)
+          const historicModel = {
+            date,
+            historicModelTemp: Number(
+              d.temperature === null ? undefined : d.temperature.toFixed(2)
```

Comment:
Why are you checking if `d.temperature` is `null`? You already filtered ones with null